examples/docs/fr-FR/table.md

```diff
@@ -1089,7 +1089,7 @@ Vous pouvez filtrer la table pour obtenir rapidement les lignes désirées.
 
 Vous pouvez customiser le contenu des colonnes afin de pouvoir utiliser d'autres composants.
 
-:::demo Vous avez acccès aux données suivantes: `row`, `column`, `$index` et `store` (gestionnaire d'état de Table) grâce aux [Scoped slot](https://vuejs.org/v2/guide/components.html#Scoped-Slots).
+:::demo Vous avez acccès aux données suivantes: `row`, `column`, `$index` et `store` (gestionnaire d'état de Table) grâce aux [Scoped slot](https://v2.vuejs.org/v2/guide/components-slots#Scoped-Slots).
```

Comment:
Note that https://v2.vuejs.org/v2/guide/components.html#Scoped-Slots will be redirect to https://v2.vuejs.org/v2/guide/components-slots.html#Scoped-Slots